packages/mdc-segmented-button/segment/styles.scss

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2020 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@use '@material/feature-targeting/feature-targeting';
+@use './segment';
+@use './segment-ripple';
+
+@mixin core-styles($query: feature-targeting.all()) {
```

Comment:
This Sass module should emit CSS.

Directly include below 2 mixins outside this mixin.

Reply:
So should `segment/` and `segmented-button/` each have a `styles.scss` file emitting css, then `./styles.scss` just includes `@use` for these `styles.scss` files?

Reply:
Child component should not have `styles.scss` file. This component should have a single entry point.

Reply:
Ok, I think I misunderstood one of your previous comments:

> Move this to `_segment-ripple.scss` file and include with and without ripple styles in `/segment/styles.scss`


So there should only be `./styles.scss` that includes the `core-styles()` and `ripple()` mixins from `segmented-button/_segmented-button.scss`, `segment/_segment.scss`, and `segment/_segment-ripple.scss`?

Reply:
Sorry, I meant `segmented-button/styles.scss` in my above comment.

yes, that's correct.